### PR TITLE
Warning fixes

### PR DESCRIPTION
**Pull request by Randl**

### CODE REVIEW

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.177-prcqbgwo completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19656788) (commit https://github.com/procxx/kepka/commit/175fa6619f by @Randl)

File: Telegram/SourceFiles/mainwidget.cpp [hunk 1]
```diff
@@ -952,28 +952,28 @@ void MainWidget::cancelUploadLayer() {
...
-					                    me->photos.removeAt(i);
-					                    MTP::send(MTPphotos_DeletePhotos(MTP_vector<MTPInputPhoto>(
-					                        1, MTP_inputPhoto(MTP_long(photo->id), MTP_long(photo->access)))));
-					                    break;
-				                    }
-			                    }
-		                    }
-	                    })));
+	Ui::show(Box<ConfirmBox>(lang(lng_delete_photo_sure), lang(lng_box_delete), base::lambda_guarded(this, [photo] {
+		                         Ui::hideLayer();
```
[the remark this answers is not in the document]
Comment (Randl) [reply Telegram/SourceFiles/mainwidget.cpp:956]: I guess it got reformatted when I removed `this`

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.179-ewfhysmb completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19675326) (commit https://github.com/procxx/kepka/commit/2cc31e0dfb by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.181-ixccxayl completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19782198) (commit https://github.com/procxx/kepka/commit/2c84107f8c by @Randl)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.181-cehgpfmw completed](https://ci.appveyor.com/project/procxx/tdesktop/builds/19819726) (commit https://github.com/procxx/kepka/commit/611f0dc82b by @Randl)

**Pull request merged**